runtime/testy/runtime.py

```diff
@@ -191,38 +201,10 @@ def runtimeTest(testNames):
 
       allProcesses.clear()
 
-      badThingsQueue = multiprocessing.Queue()
-      stateQueue = multiprocessing.Queue()
-      spawnProcess = processFactory(badThingsQueue, stateQueue, sys.stdout)
-      restartCount = 0
-      emergency_stopped = False
-
-      try:
-        spawnProcess(PROCESS_NAMES.STATE_MANAGER, startStateManager)
-        spawnProcess(PROCESS_NAMES.HIBIKE, startHibike)
-        while True:
-          if restartCount >= 3 or emergency_stopped:
-            break
-          spawnProcess(PROCESS_NAMES.STUDENT_CODE, runStudentCode, testName, 3)
-          while True:
-            try:
-              newBadThing = badThingsQueue.get(block=True)
-              print(newBadThing.event)
-              if newBadThing.event in restartEvents:
-                if(not emergency_stopped and newBadThing.event is BAD_EVENTS.EMERGENCY_STOP):
-                  emergency_stopped = True # somehow kill student code using other method? right now just restarting on e-stop
-                break
-            except Exception as e:
-              print(e)
-          stateQueue.put([SM_COMMANDS.RESET, []])
-          terminate_process(allProcesses[PROCESS_NAMES.STUDENT_CODE])
-          restartCount += 1
-        print("Funtime Runtime is done having fun.")
-        print("TERMINATING")
-      except Exception as e:
-        print("Funtime Runtime Had Too Much Fun")
-        print(e)
-        print("".join(traceback.format_tb(sys.exc_info()[2])))
+      runtime(testName)
+
+      terminate_process(PROCESS_NAMES.UDP_SEND_PROCESS)
```

Comment:
I think you want a comment about why these two terminate lines are here. (I can see that it's due to having multiple tests, but it took me a bit of cross-referencing other chunks of code to figure that out)